spec/views/teams/show.html.erb_spec.rb

```diff
@@ -11,4 +11,34 @@
     expect(rendered).to have_content(@team.kind_of_sport, count: 1)
     expect(rendered).to have_content(@team.description, count: 1)
   end
+
+  it " shows assign ownership button for mere members" do
+    user = FactoryBot.create :user    
+    @team.members << user
+    render
+    rendered.should have_content('Als Captain hinzufügen')
```

Comment:
Please use internationalization-compatible string matching instead of the hardcoded `'Als Captain hinzufügen'`! There are some examples in the `dev`'s existing codebase I think.